src/js/common/helpers/login-helpers.js

```diff
@@ -39,6 +39,8 @@ export function getUserData(dispatch) {
       dispatch(updateProfileField('status', json.data.attributes.va_profile.status));
       dispatch(updateProfileField('services', json.data.attributes.services));
       dispatch(updateLoggedInStatus(true));
+    } else {
+      dispatch(profileLoadingFinished());
```

Comment:
The only thing I'm a little confused about is when this `else` condition gets hit. Looks like when `!json.data`? When is that? Not sure I totally understand how fetch works in this case.